answers/03-modules.js

```diff
@@ -0,0 +1,21 @@
+// CommonJS, every file is module (by default)
+// Modules- encapsulated code (only share minimum)
+// const secret = 'SUPER SECRET';
+// const john = 'John';
+// const peter = 'Peter';
+// const sayHi =(name) => {
+//     console.log(`Hello there ${name}`)
+// };
+//the names variable will access the module where the shareable variables are 
+//you always start your module ./ depending on where the module is located then you might need to add . in order to direct the number of levels you need to go back
+
+const names = require('./4-names')
+const sayHi = require('./5-utils')
```

Comment:
```suggestion
const names = require('./04-names')
const sayHi = require('./05-utils')
```

These need a leading zero, otherwise the names are not the same. Remember, these are strings, not numbers. '04 !=== '4'